src/C0_data_processing.py

```diff
@@ -722,6 +810,9 @@ def define_source(dict_values, asset_name, price, output_bus, timeseries, **kwar
     else:
         source.update({"optimizeCap": {"value": False, "unit": "bool"}})
 
+    # add the parameter "maximumCap" to DSO source
+    source.update({"maximumCap": {"value": None, "unit": "kWp"}})
```

Comment:
not correct unit

Reply:
what's the correct unit here? there is no row "unit" in energyProviders, so I have to write the unit hardcoded in this cas